beta/asset-manifest.json

```diff
@@ -1,21 +1,20 @@
 {
   "files": {
     "main.css": "/beta/static/css/main.bab0b925.chunk.css",
-    "main.js": "/beta/static/js/main.5c7971d8.chunk.js",
-    "main.js.map": "/beta/static/js/main.5c7971d8.chunk.js.map",
+    "main.js": "/beta/static/js/main.4af69b38.chunk.js",
+    "main.js.map": "/beta/static/js/main.4af69b38.chunk.js.map",
```

Comment:
are we supposed to be building each pr? just wondering

Reply:
Yea, I would re-build each PR as we need to test this on the docker harness since the prod build is actually what the user will see. However, we can just build on the PRs that will get merged with the original repo.